test_pytest_pylint.py

```diff
@@ -98,9 +112,21 @@ def test_pylintrc_file_beside_ini(testdir):
     assert 'Line too long (10/3)' in result.stdout.str()
 
 
-def test_pylintrc_ignore(testdir):
+@pytest.mark.parametrize("rcformat", ("ini", "toml", "simple_toml"))
+def test_pylintrc_ignore(testdir, rcformat):
     """Verify that a pylintrc file with ignores will work."""
-    rcfile = testdir.makefile('rc', """
+    if rcformat == "toml":
+        rcfile = testdir.makefile('toml', """
+[tool.pylint.master]
+ignore = ["test_pylintrc_ignore.py", "foo.py"]
+""")
+    elif rcformat == "simple_toml":
+        rcfile = testdir.makefile('toml', """
+[tool.pylint.MASTER]
+ignore = "test_pylintrc_ignore.py,foo.py"
```

Comment:
toml has support for lists, see a few lines above.  pylint however does not support them yet.  See https://github.com/PyCQA/pylint/issues/617#issuecomment-541382799 and following comments.

Since pytest-pylint handles the ignores, we can use lists, but should also support the old format to keep switches from pylint to pytest-pylint and back simple.

`rcformat` `toml` vs `simple_toml` also tests `[tool.pylint.master]` vs `[tool.pylint.MASTER]`.  pylint supports both, see https://github.com/PyCQA/pylint/issues/617#issuecomment-541474172.